packages/vite/src/node/plugins/css.ts

```diff
@@ -885,38 +924,61 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
         })
       }
 
-      function extractCss() {
+      async function extractCss() {
         let css = ''
         const collected = new Set<OutputChunk>()
+        const concatCssEndLines: Array<{ file: string; end: number }> = []
         const prelimaryNameToChunkMap = new Map(
           Object.values(bundle)
             .filter((chunk): chunk is OutputChunk => chunk.type === 'chunk')
             .map((chunk) => [chunk.preliminaryFileName, chunk]),
         )
 
+        let line = 1
+
         function collect(fileName: string) {
           const chunk = bundle[fileName]
-          if (!chunk || chunk.type !== 'chunk' || collected.has(chunk)) return
+          if (!chunk || chunk.type !== 'chunk' || collected.has(chunk)) {
+            return
+          }
           collected.add(chunk)
 
           chunk.imports.forEach(collect)
-          css += chunkCSSMap.get(chunk.preliminaryFileName) ?? ''
+
+          const content = chunkCSSMap.get(chunk.preliminaryFileName) ?? null
+          if (content == null) {
+            return
+          }
+          if (css !== '') {
+            css += '\n' + content
```

Comment:
Same here
```suggestion
            if (content !== '') {
              css += '\n' + content
            }
```

Reply:
For the concating content, I have changed it to return when the `content` is `''`.

For not pushing the content, the same conditional checks if the content is `null`, `''` or not.
So, I can confirm that this content won't get pushed to the `concatCssEndLines`.